src/microbe_stage/Membrane.cs

```diff
@@ -370,9 +389,12 @@ private void ApplyTextures()
         albedoTexture = Type.LoadedAlbedoTexture;
 
         // This is called rarely enough that this just plain re-creates StringName instances here each time
-        MembraneShaderMaterial!.SetShaderParameter("albedoTexture", albedoTexture);
-        MembraneShaderMaterial.SetShaderParameter("normalTexture", Type.LoadedNormalTexture);
-        MembraneShaderMaterial.SetShaderParameter("damagedTexture", Type.LoadedDamagedTexture);
+        if (MembraneShaderMaterial != null)
```

Comment:
What's the reason why this code is required now? If this kind of error checking is needed then a bigger refactor is needed as this would just *silently ignore* applying a material and pretend that it applied correctly, which is not the right way to do error handling.

Reply:
I'll start doing work on this branch to finish off these comments that have not been addressed (as far as I can tell only master has been merged into this branch recently which is useless for everything else except spamming me for emails, a branch that doesn't have all review comments solved *does not need to be in good merge state* as it cannot be merged anyway due to the review comments).